tailwind.config.js

```diff
@@ -3,30 +3,77 @@
 const defaultTheme = require('tailwindcss/defaultTheme')
 
 module.exports = {
+  darkMode: 'class',
   content: ['./index.html', './src/**/*.{vue, js, ts}'],
+  plugins: [
+    require('@gradin/tailwindcss-scrollbar')
+  ],
   theme: {
     extend: {
       fontFamily: {
         sans: ['Plus Jakarta Sans', ...defaultTheme.fontFamily.sans]
       },
       colors: {
         'main-purple': '#635FC7',
+        'transparent-purple': '#635FC71A',
+        'semitransparent-purple': ' #635FC740',
+        'white-purple': '#635FC71A',
         'main-purple-hover': '#A8A4FF',
         'regular-black': '#000112',
         'very-dark-grey': '#20212C',
         'dark-grey': '#2B2C37',
         'lines-dark': '#3E3F4E',
-        'medium-grey': '828FA3',
+        'medium-grey': '#828FA3',
         'lines-light': '#E4EBFA',
         'light-grey': '#F4F7FD',
         'regular-red': '#EA5555',
-        'red-hover': '#FF9898'
+        'red-hover': '#FF9898',
+        'first-column': '#49C4E5',
```

Comment:
I'd go with tailwinds style of color name, so this would be `blue: { 400: "#49C4E5"}`.

What if we'd need to use this color on some border? 
It seems weird to set border color as `border-first-column`...

Reply:
Good idea, **corrected**.